### ISSUE TITLE
bazel version reports incorrect build time

### ISSUE THREAD

**Issue by ImJasonH** [bazelbuild/bazel#4936]
### Description of the problem / feature request:

`bazel version` (running 0.11.1) reports "Build time" as `Tue May 14 07:48:23 +50148 (1520362424903)`

The timestamp appears to be milliseconds (1520362424 is March 6, 2018), but is being interpreted by `bazel version` as seconds, which is May 14, 50148.

Bazel is awesomely futuristic, but I doubt it's actually built forty-eight thousand years in the future and sent back to us in some kind of time machine :) 

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

```
$ bazel version
.............
Build label: 0.11.1
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Tue May 14 07:48:23 +50148 (1520362424903)
Build timestamp: 1520362424903
Build timestamp as int: 1520362424903
```

### What operating system are you running Bazel on?

Linux

### What's the output of `bazel info release`?

```
$ bazel info release
release 0.11.1
```

**Comment by deeglaze** [comment]
This is a dup of my issue [#4905](https://github.com/bazelbuild/bazel/issues/4905) which is again a dup of [#4469](https://github.com/bazelbuild/bazel/issues/4469) :(

**Comment by vladmos** [comment]
I'll close the issue as it's a duplicate, also there's a recent commit that's supposed to fix the issue: e52c8418d37e902237c6676d06fcb4cad7fc4766
